src/main/scala/com/gu/mobile/content/notifications/lib/ContentAlertPayloadBuilder.scala

```diff
@@ -77,11 +83,11 @@ trait ContentAlertPayloadBuilder extends Logging {
     getTopicType(tag.`type`) map { tagType => Topic(tagType, tag.id) }
   }
 
-  private def contentTitle(followableTag: Option[Tag], topics: List[Topic]): Option[String] =
+  private def contentTitle(followableTag: List[Tag], topics: List[Topic]): Option[String] =
     if (topics.toSet.intersect(topicsWithoutPrefix).nonEmpty) {
       None
     } else {
-      followableTag.map { ft => ft.webTitle }.orElse(Some("Following"))
+      Some(followableTag.map { ft => ft.webTitle }.mkString(","))
```

Comment:
I think you probably want a space after that comma. Which mean you'll need to adapt your test too

Reply:
I did have a space initially, wasn't sure on correct format. I will change now 👍